specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2021-04-30-preview/iotcentral.json

```diff
@@ -1995,7 +2230,7 @@
         }
       },
       "post": {
-        "operationId": "Devices_ExecuteModuleCommand",
+        "operationId": "Devices_RunModuleCommand",
         "summary": "Execute a module command",
```

Comment:
Please update the summary and description (Execute -> Run) too.

Reply:
Fixed comments as well.